Source/Scene/Cesium3DTileStyle.js

```diff
@@ -0,0 +1,451 @@
+         * <p>
+         * The expression must return or convert to a <code>Boolean</code>.
+         * </p>
+         *
+         * @memberof Cesium3DTileStyle.prototype
+         *
+         * @type {StyleExpression}
+         *
+         * @exception {DeveloperError} The style is not loaded.  Use Cesium3DTileStyle.readyPromise or wait for Cesium3DTileStyle.ready to be true.
+         *
+         * @example
+         * var style = new Cesium3DTileStyle({
+         *     show : '(regExp("^Chest").test(${County})) && (${YearBuilt} >= 1970)'
+         * });
+         * style.show.evaluate(frameState, feature); // returns true or false depending on the feature's properties
+         *
+         * @example
+         * var style = new Cesium.Cesium3DTileStyle();
+         * // Override show expression with a custom function
+         * style.show = {
+         *     evaluate : function(frameState, feature) {
+         *         return true;
+         *     }
+         * };
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+         */
+        show : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this._ready) {
+                    throw new DeveloperError('The style is not loaded.  Use Cesium3DTileStyle.readyPromise or wait for Cesium3DTileStyle.ready to be true.');
+                }
+                //>>includeEnd('debug');
+
+                return this._show;
+            },
+            set : function(value) {
+                this._showShaderFunctionReady = false;
+                this._show = value;
+            }
+        },
+
+        /**
+         * Gets or sets the {@link StyleExpression} object used to evaluate the style's <code>color</code> property.
+         * <p>
+         * The expression must return a <code>Color</code>.
+         * </p>
+         *
+         * @memberof Cesium3DTileStyle.prototype
+         *
+         * @type {StyleExpression}
+         *
+         * @exception {DeveloperError} The style is not loaded.  Use Cesium3DTileStyle.readyPromise or wait for Cesium3DTileStyle.ready to be true.
+         *
+         * @example
+         * var style = new Cesium3DTileStyle({
+         *     color : '(${Temperature} > 90) ? color("red") : color("white")'
+         * });
+         * style.color.evaluateColor(frameState, feature, result); // returns a Cesium.Color object
+         *
+         * @example
+         * var style = new Cesium.Cesium3DTileStyle();
+         * // Override color expression with a custom function
+         * style.color = {
+         *     evaluateColor : function(frameState, feature, result) {
+         *         return Cesium.Color.clone(Cesium.Color.WHITE, result);
+         *     }
+         * };
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
```

Comment:
No need to mention this more than in the top-level summary.  Looking at the compiled doc for this file, there's a lot of noise.